File: Programmers/JAVA/lv1/lv1_두 개 뽑아서 더하기.java
https://programmers.co.kr/learn/courses/30/lessons/68644

import java.util.*;

class Solution {
    public int[] solution(int[] numbers) {
        ArrayList<Integer> list = new ArrayList<>();
        int sum = 0;
        for(int i = 0; i < numbers.length; i++){
            for(int j = i + 1; j <numbers.length; j++){
                sum = numbers[i] + numbers[j];
                if(list.indexOf(sum) == -1)
                   list.add(sum);
            }
        }
        
        int[] answer = new int[list.size()];
        for(int i = 0; i < list.size(); i++){
            answer[i] = list.get(i);
        }
        Arrays.sort(answer);
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_두 개 뽑아서 더하기.cs
https://programmers.co.kr/learn/courses/30/lessons/68644

using System;
using System.Collections.Generic;

public class Solution
{
    public int[] solution(int[] numbers)
    {
        List<int> answer = new List<int>();

        int sum = 0;
        for (int i = 0; i < numbers.Length; i++)
        {
            for (int j = i + 1; j < numbers.Length; j++)
            {
                sum = numbers[i] + numbers[j];
                if (!(answer.Contains(sum)))
                    answer.Add(sum);
            }
        }
        answer.Sort();

        return answer.ToArray();
    }
}